README.md

```diff
@@ -91,10 +91,9 @@ Load relevant MathJax (v2) files *Before* the plotly.js script tag:
     <script src="https://cdn.plot.ly/plotly-2.0.0-rc.2.min.js"></script>
 ```
 
-## Partial bundles
-
-There are two kinds of plotly.js partial bundles:
-1. The official partial bundles that are distributed to `npm` and the CDN, described in [the dist README](https://github.com/plotly/plotly.js/blob/master/dist/README.md).
+## Bundles
+There are two kinds of plotly.js bundles:
+1. Complete and partial official bundles that are distributed to `npm` and the `CDN`, described in [the dist README](https://github.com/plotly/plotly.js/blob/master/dist/README.md).
 2. Custom bundles you can create yourself, if none of the distributed packages meet your needs.
```

Comment:
I think this custom-bundles section could live in its own separate file, which would be linked to from here (and then all three of dist/custom-bundles/building readmes would link to each other as they are essentially alternative ways of doing similar things)